1.5.1

```diff
@@ -0,0 +1,83 @@
+{
+    "instructionList": [
+        {
+            "sObjectName": "Activity",
+            "taskList": [
+                {
+                    "jiraTicket": "AC-24873a",
+                    "actionType": "RenameStandardFieldLabel",
+                    "actionTypeName": "type_activity",
+                    "description": "Update the standard field label",
+                    "actionTypeLabel": "Meeting Type"
+                }
+            ]
+        },
+        {
+            "taskList": [
+                {
+                    "jiraTicket": "AC-24873b",
+                    "actionType": "DeclarativeMetadata",
+                    "actionTypeName": "dummy",
+                    "description": "Update the standard field label on the event object",
+                    "optionsMap": {
+                        "filePath": "objectTranslations",
+                        "packageMember": "CustomObjectTranslation",
+                        "packageSubMembers": "Activity-en_US",
+                        "labelUpdate": "true"
+                    }
+                }
+            ]
+        },
+        {
+            "sObjectName": "Event",
+            "description": "Add new picklist values to Type",
+            "taskList": [
+                {
+                    "jiraTicket": "ac-24873c",
+                    "description": "Path to Deactivate the existing value",
+                    "actionType": "PicklistValueStandard",
+                    "actionTypeName": "Type",
+                    "optionsMap": {
+                        "picklistValues": "Email,Meeting,Other,Call",
+                        "picklistOperation": "deactivate"
+                    }
+                }
+            ]
+        },
+        {
+            "sObjectName": "Event",
+            "description": "Add new picklist values to Type",
+            "taskList": [
+                {
+                    "jiraTicket": "ac-24873d",
+                    "description": "Path to add 1st Meeting,2nd Meeting,3rd Meeting,Delivery Meeting,Annual Review values to Type",
+                    "actionType": "PicklistValueStandard",
+                    "actionTypeName": "Type",
+                    "optionsMap": {
+                        "picklistValues": "1st Meeting,2nd Meeting,3rd Meeting,Delivery Meeting,Annual Review",
+                        "picklistOperation": "add"
+                    }
+                }
```

Comment:
Is it possible to just combine this into the 'c' task above? Or did you have a reason to keep it separate?

Reply:
We can combine these and updated the instruction.